### PR TITLE
[SoftSign-9][PYTHON] Python API for SoftSign

### PR DESCRIPTION
**Pull request by jiwaszki**
### Details:
 - Added SoftSign to OV Python API in opset9 (both current and compatibility).
 - Added tests. Upgrade of opset version in some files.
 
### Tickets:
 - *79226*

### CODE REVIEW

File: src/bindings/python/tests/test_ngraph/test_create_op.py [hunk 1]
```diff
@@ -1962,3 +1962,14 @@ def test_nv12_to_rgb():
...
+
+def test_softsign():
+    input_shape = [2, 4, 8, 16]
+
+    param = ov.parameter(input_shape, name="input")
+    node = ov.softsign(param, input_shape)
+
+    assert node.get_type_name() == "SoftSign"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == input_shape
```
Comment (mitruska) [inline src/bindings/python/tests/test_ngraph/test_create_op.py:1975]: Please add assert for `node.get_output_element_type(0)`

File: src/bindings/python/tests_compatibility/test_ngraph/test_create_op.py [hunk 2]
```diff
@@ -2028,3 +2028,14 @@ def test_nv12_to_rgb():
...
+
+def test_softsign():
+    input_shape = [2, 4, 8, 16]
+
+    param = ng.parameter(input_shape, name="input")
+    node = ng.softsign(param, input_shape)
+
+    assert node.get_type_name() == "SoftSign"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == input_shape
```
Comment (mitruska) [inline src/bindings/python/tests_compatibility/test_ngraph/test_create_op.py:2041]: Please add assert for `node.get_output_element_type(0)`

File: src/bindings/python/tests/test_ngraph/test_create_op.py [hunk 3]
```diff
@@ -1962,3 +1962,14 @@ def test_nv12_to_rgb():
     assert node_separate_planes.get_output_size() == 1
     assert node_separate_planes.get_output_element_type(0) == Type.f32
     assert list(node_separate_planes.get_output_shape(0)) == expected_output_shape
+
+
+def test_softsign():
+    input_shape = [2, 4, 8, 16]
+
+    param = ov.parameter(input_shape, name="input")
```
Comment (mitruska) [inline src/bindings/python/tests/test_ngraph/test_create_op.py:1970]: I'm a little bit surprised, that it is possible to create `parameter` without specifying `element_type`. 
What is set by default?

File: src/bindings/python/src/compatibility/ngraph/opset9/ops.py [hunk 4]
```diff
@@ -46,3 +46,14 @@ def eye(
         inputs = as_nodes(num_rows, num_columns, diagonal_index)
 
     return _get_node_factory_opset9().create("Eye", inputs, {"output_type": output_type})
+
+
+@nameable_op
+def softsign(node: NodeInput, name: Optional[str] = None) -> Node:
+    """Apply SoftSign operation on the input node element-wise.
+
+    :param note: One of: input node, array or scalar.
```
Comment (mitruska) [inline src/bindings/python/src/compatibility/ngraph/opset9/ops.py:55]:
```suggestion
    :param node: One of: input node, array or scalar.
```

File: src/bindings/python/src/openvino/runtime/opset9/ops.py [hunk 5]
```diff
@@ -46,3 +46,14 @@ def eye(
         inputs = as_nodes(num_rows, num_columns, diagonal_index)
 
     return _get_node_factory_opset9().create("Eye", inputs, {"output_type": output_type})
+
+
+@nameable_op
+def softsign(node: NodeInput, name: Optional[str] = None) -> Node:
+    """Apply SoftSign operation on the input node element-wise.
+
+    :param note: One of: input node, array or scalar.
```
Comment (mitruska) [inline src/bindings/python/src/openvino/runtime/opset9/ops.py:55]:
```suggestion
    :param node: One of: input node, array or scalar.
```

File: src/bindings/python/tests/test_ngraph/test_create_op.py (hunk 3 above)
Comment (jiwaszki) [reply src/bindings/python/tests/test_ngraph/test_create_op.py:1970]: f32 (or if you want to keep it pythonic np.float32) - this is just a part of friendly PyAPI:
https://github.com/openvinotoolkit/openvino/blob/b1ecd3ea8f10b057cd592e72e56b88efc401095d/src/bindings/python/src/openvino/runtime/opset1/ops.py#L1797-L1799

Comment (mitruska) [reply src/bindings/python/tests/test_ngraph/test_create_op.py:1970]: Ok

**Review by mitruska — APPROVED** [review]

**Pull request merged**